### DISCUSSION TITLE
Pod relationship data gets saved incomplete

### DISCUSSION THREAD

**Discussion by pcfreak30** [pods-framework/pods#6635] [Q&A]
First i am unsure as to how this is possible right now but I have only tracked the result.

![image](https://user-images.githubusercontent.com/10647/30370700-5c62c32e-9846-11e7-9ebc-c829b4a04d04.png)

Somehow there is a situation where the related_pod_id and related_field_id could become 0. I doubt this will be figured out any time soon but I thought it should be documented.

For now the solution is to just re save the item in question.

**Comment by sc0ttkclark** [comment]
Are you sure this is a bidirectional relationship? Those fields are **only** used by bidirectional relationships.

**Comment by pcfreak30** [comment]
Yes it is.

**Comment by sc0ttkclark** [comment]
Bidirectional info gets pulled in here before saving:
https://github.com/pods-framework/pods/blob/2.x/classes/PodsAPI.php#L4001
